dist/index.html

```diff
@@ -4,12 +4,10 @@
     <meta charset="utf-8">
     <title>BloodStone Bookmark Manager</title>
     <script src="bundle.js"></script>
-  <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700" rel="stylesheet">
```

Comment:
merge the 2 font links: `https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700|Work+Sans`